.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Docker Publishing
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+
+jobs:
+  publish:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Publish to Registry
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.pcicdevops_at_dockerhub_username }}
+          password: ${{ secrets.pcicdevops_at_dockerhub_password }}
+          dockerfile: ./dev.Dockerfile
+          repository: pcic/pycds
+          tag_with_ref: true
+
+  publish-local:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Publish to Registry
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.pcicdevops_at_dockerhub_username }}
+          password: ${{ secrets.pcicdevops_at_dockerhub_password }}
+          dockerfile: ./docker/local-test/Dockerfile
+          repository: pcic/pycds
+          tags: local-test
```

Comment:
Argh, a bit of an oversight here on my part when you created the docker-publish action. Yes, I do want to remove `dev.Dockerfile`. I was thinking there was also a regular, "production" Dockerfile, which of course there is not because this is a library package, not an application.

We do need the local-test dockerfile etc., but it is only for establishing a test environment. Do you think we should, as I have seen in other repos, make the Dockerfile at the root of the project which is a link to the one in `docker/...`? I think that makes the most sense but I am happy to follow whatever practices you prefer.

Reply:
That makes the most sense, moving it would be preferable!